CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Fixed
 
+- 🐛 Fixed a bug on macOS, caused by accessing an invalidated iterator after erasure in a loop ([#932](https://github.com/munich-quantum-toolkit/core/pull/932)) ([**@flowerthrower**](https://github.com/flowerthrower))
```

Comment:
```suggestion
- 🐛 Fix bug in MLIR roundtrip passes caused by accessing an invalidated iterator after erasure in a loop ([#932](https://github.com/munich-quantum-toolkit/core/pull/932)) ([**@flowerthrower**](https://github.com/flowerthrower))
```

See https://common-changelog.org/#241-change for some guidelines on how to write changelog entries.
The previous entry provided no context for people reading the changelog on what kind of bug was fixed.